test/config-message-function.js

```diff
@@ -472,4 +472,19 @@ describe('config: message function', function() {
       done();
     }
   });
+
+  it('can change GULPLOG_DEPRECATED with .gulp.*', function(done) {
+    var cwd = path.join(baseDir, 'GULPLOG_DEPRECATED');
+    var expected = 'GULPLOG V1 IS DEPRECATED\n';
+
+    var opts = { cwd: cwd };
+    exec(gulp(), opts, cb);
+
+    function cb(err, stdout, stderr) {
+      expect(err).toBeNull();
+      expect(stdout).toEqual('');
+      expect(stderr).toEqual(expected);
```

Comment:
Shouldn't the message the plugin attempted be output with deprecated message?

Reply:
I silenced all other messages via the config file. Do you think I need to allow the other message?

Reply:
I believe `gulplog-v1` is intended for messages output by existing plugins. Wouldn't it be problematic if those messages are not output?

Why isn't the original message (`"Some error here"` in this test case) being output after the deprecated message in [this part of the code](https://github.com/gulpjs/glogg/blob/master/index.js#L21-L23)?

Reply:
I think you don't understand how the translation code works. If `false` is returned, nothing is printed.

Reply:
I understood why the original error message is not being output.

If the message function returns false, is it possible all messages from existing plugins not to be output, too? Or, are there plans to control the output of plugin messages using this message function in the future?

Reply:
Any messages to gulplog can be controlled through the `message` function, as you've seen here. Generally plugin errors will just be strings, not symbols, so users will have to match string values.

Reply:
I see. However, in actual cases, most of times plugin messages will be output as they are. Therefore, I think it would be better to add test cases where the original message is output.

Reply:
This test is a unit test to ensure the deprecate message can be translated. There may be a case for creating another test to test the original message but I don't want to dig into where that can fit in the harness. This is a good enough test for now.